src/main/java/uk/gov/companieshouse/api/strikeoffobjections/interceptor/ObjectionInterceptor.java

```diff
@@ -35,6 +37,20 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
 
         try {
             Objection objection = objectionService.getObjection(requestId, objectionId);
+            
+            // Operations on objections via the API REST interface are only allowed whilst the objection is
```

Comment:
This is a ‘belt and braces’ change since removing the objection id from the web session once an objection is successfully submitted - see change in https://github.com/companieshouse/strike-off-objections-web/pull/155 - should ensure the API doesn’t receive any further REST calls for that objection. But if it did (e.g. by the API being called directly from an untrusted client) this check will stop them going through.

Reply:
what happens when we submit via Patch, won't the ObjectionStatus.OPEN != objection.getStatus() come in to play?

Reply:
hold on, we submit when open, sorry.